index.html

```diff
@@ -904,38 +904,36 @@ <h2>Privacy Considerations</h2>
     </p>
     <section>
       <h2>Persistent information</h2>
+      <p>
+        Discovery of encoder and decoder capabilities is out of scope
+        of this specification. Consistent and accurate information
```

Comment:
I would move statements of scope further up in the spec, and focus on describing the privacy considerations inherent in this spec. I'd start with the informational part, and then follow with why that might be similar to existing APIs and how this compares.

Reply:
I have removed the first sentence.